tests/migration_test.go

```diff
@@ -432,7 +434,7 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 
 			It("should complete a migration", func() {
 				vmi := tests.NewRandomFedoraVMIWitGuestAgent()
-				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
+				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512M")
```

Comment:
I think you can remove this line if you reduce the memory.

```golang
func NewRandomFedoraVMIWitGuestAgent() *v1.VirtualMachineInstance {
	agentVMI := NewRandomVMIWithEphemeralDiskAndUserdata(ContainerDiskFor(ContainerDiskFedora), GetGuestAgentUserData())
	agentVMI.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512M")
	return agentVMI
}
```

Reply:
Good catch @SchSeba, thanks.